pyproject.toml

```diff
@@ -1,8 +1,54 @@
 [build-system]
+build-backend = "mesonpy"
 requires = [
-    "setuptools<60",
-    "wheel",
-    "oldest-supported-numpy",
-    "numpy!=1.22.0",
     "mako",
+    "meson-python",
+    "oldest-supported-numpy",
+    "wheel",
+]
+
+[project]
+name = "pyfmmlib"
+version = "2023.1"
```

Comment:
```suggestion
version = "2024.1"
```

Reply:
Updated in latest version
https://github.com/inducer/pyfmmlib/compare/940dd0a659198eb9113e0cd0db975b88bb969dca..f0a48f5676ee4954828ff306ff6b8aca834b1b9a